### DISCUSSION TITLE
Raspberry Pi4

### DISCUSSION THREAD

**Discussion by chellea** [nebhead/PiTender#15] [Q&A] [answered]
I know that I am late to the discussion, but I have been following for some time with a degree of envy as I waited for the time to tackle this.  Well, that time is now!!!

Is there any reason that this could not run on a Raspberry Pi 4?  Are there any changes necessary to make that happen?

Thank you,

**Comment by nebhead** [comment, marked as answer]
Yep, nothing different about a Pi4 in this case.  Should work great!  Enjoy!

**Reply by chellea** [reply]
Excellent!  I began my "rough" build today, will be loading up the pi in the morning.  Once I test everything, I will build it's final enclosure and post pictures.  

Thank you
